SystemManager/Src/SM.cpp

```diff
@@ -0,0 +1,135 @@
+/*
+ * SM.cpp
+ *
+ * System Manager State-Machine classes
+ *
+ * Created on: Nov 7, 2022
+ * Author(s): Gordon Fountain
+ */
+
+#include "SM.hpp"
+
+#include "SM_States.hpp"
+#include "cmsis_os.h"
+#include "task.h"
+
+namespace SM {
+
+uint32_t AM_PERIOD_MS = 5;  // Current operation speed of 200 Hz.
+
+/*
+Helper Functions
+*/
+
+AM::AttitudeManagerInput RcToAmInput(LosLinkRx_t rc_message)
```

Comment:
This might need to change, depending on what we want to do for flight test :/

Reply:
I know. If we need I can build one for quad too, but I haven't been asked yet.